connectorx-python/src/pandas/mysql_pandas.rs

```diff
@@ -0,0 +1,55 @@
+use super::destination::PandasDestination;
+use super::types::PandasTypeSystem;
+use connectorx::{
+    impl_transport,
+    sources::mysql::{MysqlSource, MysqlTypeSystem},
+    typesystem::TypeConversion,
+};
+use rust_decimal::prelude::*;
+use uuid::Uuid;
+use chrono::{DateTime, NaiveDate, NaiveTime, NaiveDateTime, Utc};
+
+pub struct MysqlPandasTransport<'py>(&'py ());
+
+impl_transport!(
+    name = MysqlPandasTransport<'tp>,
+    systems = MysqlTypeSystem => PandasTypeSystem,
+    route = MysqlSource => PandasDestination<'tp>,
+    mappings = {
+        { Double[f64]                => F64[f64]                | conversion all }
+        { Long[i64]                  => I64[i64]                | conversion all }
+        { Date[NaiveDate]            => DateTime[DateTime<Utc>] | conversion half }
+        { Time[NaiveTime]            => String[String]          | conversion half }
+        { Datetime[NaiveDateTime]    => DateTime[DateTime<Utc>] | conversion half }
+        { Decimal[Decimal]           => F64[f64]                | conversion half }
+        // { Char[&'r str]              => Char[&'r str]           | conversion none }
+        // { VarChar[&'r str]           => Str[&'r str]            | conversion none }
+        { Char[String]              => Char[String]           | conversion none }
```

Comment:
seems like the destination type does not match?
Char -> char
Str -> &str (maybe we should use String[String] here)